### PR TITLE
[typescript-fetch] support custom stringify for query string

### PR DESCRIPTION
**Pull request by hello-brsd**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`. If contributing template-only or documentation-only changes which will change sample output, be sure to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) first.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.1.x`, `5.0.x`. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

It would be helpful if the query string formatting could be configured.
Solution for example for the deep query string:
```
import { Configuration, PetApi } from './api'
import { stringify } from 'qs'

const config = new Configuration({ queryParamsStringify: stringify })
const api = new PetApi(config)
```

@TiFu (2017/07) @taxpon (2017/07) @sebastianhaas (2017/07) @kenisteward (2017/07) @Vrolijkx (2017/09) @macjohnny (2018/01) @nicokoenig (2018/09) @topce (2018/10) @akehir (2019/07)

### CODE REVIEW

File: modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache [hunk 1]
```diff
@@ -116,6 +116,7 @@ export interface ConfigurationParameters {
     basePath?: string; // override base path
     fetchApi?: FetchAPI; // override for fetch implementation
     middleware?: Middleware[]; // middleware to apply before/after fetch requests
+    stringify?: (params: HTTPQuery) => string; // stringify function for query strings
```
Comment (macjohnny) [inline modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache]: naming should be more specific, as it is only used for query parameter serialization:
```suggestion
    queryParamsStringify?: (params: HTTPQuery) => string; // stringify function for query strings
```

Comment (hello-brsd) [reply modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache]: thank you for the review, i renamed it.

**Comment by akehir** [comment]
> LGTM

Thumbs up.

**Comment by macjohnny** [comment]
circle ci failure is unrelated

**Comment by wing328** [comment]
@hello-brsd thanks for the PR, which has been included in the 4.1.0 release: https://twitter.com/oas_generator/status/1160000504455319553

**Pull request merged**